src/tui/proc-table.go

```diff
@@ -46,6 +47,12 @@ func (pv *pcView) fillTableData() {
 			pv.procTable.RemoveRow(row)
 			continue
 		}
+
+		if pv.processRegex != nil && !pv.processRegex.MatchString(state.Name) {
```

Comment:
I think `pv.processRegex` should be protected with a mutex.
It's being read and written from 2 different threads.